cvat/apps/engine/utils.py

```diff
@@ -210,7 +210,7 @@ def get_rq_lock_by_user(queue: DjangoRQ, user_id: int, *, timeout: Optional[int]
         )
     return nullcontext()
 
-def get_rq_lock_for_job(queue: DjangoRQ, rq_id: str, *, timeout: Optional[int] = 60, blocking_timeout: Optional[int] = None) -> Lock:
+def get_rq_lock_for_job(queue: DjangoRQ, rq_id: str, *, timeout: Optional[int] = 60, blocking_timeout: Optional[int] = 60) -> Lock:
```

Comment:
Shouldn't `blocking_timeout` be less than the request timeout? (to minimize 504 responses)